Felienne/hedy
# [FEATURE] Enables teachers to create accounts

**Description**
In this PR we implement a new feature, enabling teachers to easily create multiple accounts for their students. Instead of each students having to fill out their own sign up form the teacher can create unlimited account in one go. We implement a new section on the `/for-teachers` page with a "create accounts" button. Directing the teacher to a dedicated "create multiple accounts" page. The teacher only has to fill in the username, mailadres and password for each student. If desired they can also automatically be assigned to a class.

**One important issue to solve**
What do we do with in-active accounts? I can imagine a teacher creating multiple accounts for which some are never used. Do we implemented some timer, that if not used within a week they are automatically removed?

**Fix for**
This PR fixes #1386.

**How to test**
Make sure you are a teacher and navigate to the `/for-teachers` page. Notice the new section with the "create accounts" button. Click it and try to add multiple accounts. Try different combinations and make sure that each field is required. Also try adding an already existing username or mail address. This should result in a warning.

**Important**
This requires some dedicated testing to make sure we don't break anything. Adding multiple accounts add once is a heavy load on the database due to the checking and verifying of each unique username / mail address, the existence of the class and the actual storing of the new user.
